app/controllers/traces_controller.rb

```diff
@@ -33,7 +34,7 @@ def index
              elsif current_user && current_user == target_user
                t ".my_gps_traces"
              else
-               t ".public_traces_from", :user => target_user.display_name
+               t ".public_traces_from_html", :user => tag.bdi(target_user.display_name)
```

Comment:
Doesn't work as expected for some reason. That's what I see in the upper left corner on `/user/:name/traces` pages:

![image](https://github.com/user-attachments/assets/e69c44e3-cb62-4e22-8409-f9ce59e2594b)

Reply:
So I've failed to find out exactly what's going on here but something deep in the rails output capture magic is going wrong and causing most of the output inside the `tag.head` block in `app/views/layouts/_head.html.erb` to be output directly instead of being captured and then wrapped in the tag.

It turns out that `bdi` tags in a title element don't work anyway though as the browser will just render them as text in the title bar, so I've replaced them with the equivalent unicode characters which is a trick we already use in `set_title`.